packages/kbn-ftr-common-functional-ui-services/services/remote/webdriver.ts

```diff
@@ -232,7 +232,7 @@ async function attemptToCreateCommand(
 
         if (headlessBrowser === '1') {
           // See: https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Headless_mode
```

Comment:
This URL doesn't seem to work anymore. The closest I could find was https://developer.mozilla.org/en-US/docs/Web/WebDriver/Capabilities/firefoxOptions or https://hacks.mozilla.org/2017/12/using-headless-mode-in-firefox/ - might be worth updating?

Reply:
I pushed b78c7a99bbc6a5fce8a9d51fa498b59ccaecf160